src/db/rss-parser.ts

```diff
@@ -0,0 +1,45 @@
+import Parser from 'rss-parser';
+import * as publications from '../../publications.json';
+
+import { Article } from '../entities/Article';
+
+const getRecentArticles = async (): Promise<Article[]> => {
+  const parser = new Parser();
+
+  const publicationsDB = publications.publications;
+  const nameToSlugMap = {};
+  const feedSources = publicationsDB.map((publication) => {
+    nameToSlugMap[publication['rss-name']] = publication.slug;
+    return publication.feed;
+  });
+
+  const feedRequests = feedSources.map((feed) => {
+    return parser.parseURL(feed);
+  });
+
+  const articlePromises = Promise.all(feedRequests).then((feeds) => {
+    const articles = new Array<Article>();
```

Comment:
Mostly stylistic suggestion: if you are using node >= 11 (current lts is 14), you could use [Array.prototype.flat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat) on the result of your nested maps instead of continuously mutating the `articles` variable (the `const` declaration of which is misleading if you ask me).

Reply:
fixed